bool geofence_violated		# true if the geofence is violated
